dummy/dummy.js

```diff
@@ -0,0 +1,127 @@
+container - HTMLDivElement {
```

Comment:
@lpham598 do we still need this file? If not can we delete it?

Reply:
I think Rudo wanted to look at the structure of the root fiber.

Reply:
Then can we change the name to TicTacToefiberNode and put in MockData? or where it makes sense...lets not keep it as dummy/dummy.js that would not mean anything ...